_layouts/home.html

```diff
@@ -0,0 +1,227 @@
+    <div class="container">
+        <div class="row">
+            <div class="col-lg-11 col-md-12 ml-auto">
+                <h2 class="section-title bar-right"><b>WHAT IS Vac</b></h2>
+            </div>
+        </div>
+        <div class="row">
+            <div class="col-lg-5 col-md-6 ml-auto mb-60">
+                <p>Vac is being designed for client developers to provide the highest level of privacy and security for their users.</p>
+            </div>
+            <div class="col-lg-4 col-md-6 ml-auto mr-auto">
+                <div class="btn-holder">
+                    <a href="https://github.com/vacp2p" target="_blank" class="btn btn-purple">Learn More</a>
+                </div>
+            </div>
+        </div>
+    </div>
+    <div class="container">
+        <div class="features swiper-container">
+            <div class="swiper-wrapper">
+                <div class="swiper-slide">
+                    <div class="row">
+                        <div class="col-lg-5 col-md-6 ml-auto">
+                            <img src="{{ '/assets/img/feature_1.svg' | relative_url }}">
+                        </div>
+                        <div class="col-lg-5 col-md-6 mr-auto d-flex justify-content-center flex-column">
+                            <h3>Full messaging stack</h3>
+                            <p>Concerned with all layers in the stack including underlying transports, p2p overlays and routing, initial trust establishment, and semantics for things like group chat.</p>
+                        </div>
+                    </div>
+                </div>
+                <div class="swiper-slide">
+                    <div class="row">
+                        <div class="col-lg-5 col-md-6 ml-auto">
+                            <img src="{{ '/assets/img/feature_2.svg' | relative_url }}">
+                        </div>
+                        <div class="col-lg-5 col-md-6 mr-auto d-flex justify-content-center flex-column">
+                            <h3>Modular system</h3>
+                            <p>Provide options at each layer in the stack, instead of having a tightly coupled set of protocols. Allowing developers to choose what they use and ensure they know each choice comes with different trade-offs.</p>
+                        </div>
+                    </div>
+                </div>
+                <div class="swiper-slide">
+                    <div class="row">
+                        <div class="col-lg-5 col-md-6 ml-auto">
+                            <img src="{{ '/assets/img/feature_3.svg' | relative_url }}">
+                        </div>
+                        <div class="col-lg-5 col-md-6 mr-auto d-flex justify-content-center flex-column">
+                            <h3>Peer-to-Peer</h3>
+                            <p>The protocols we work on are pure p2p, and  aim to minimize centralization. This too is in opposition to many initiatives in the secure messaging space.</p>
```

Comment:
lets replace 'pure p2p' with 'peer-to-peer'

Reply:
Done